src/donation/models/donation-type.model.ts

```diff
@@ -0,0 +1,4 @@
+export enum DonationType {
```

Comment:
I believe we're missing 'red blood cell concentrate' and 'platelet concentrate'?

Reply:
We will add a few more most likely. If you can provide me with "for sure" instead of "I believe" I'll add these or I can add them later on as it's a rather simple change.
(this comment felt agressive when typing, it's not :D)

Reply:
Based on [this website](https://www.vitalant.org/types-of-donation), I'd add:

- `DOUBLE_RED_CELL`
- `PLATELET`

Reply:
Since both are viable I'm going for `power` instead of `double_red` because it sounds cooler :D